tasks/marketplacesvm-push-disk-images/tests/test-marketplacesvm-push-disk-images.yaml

```diff
@@ -0,0 +1,172 @@
+                      }
+                    }
+                    "staged": {
+                      "destination": "test-product-1_DOT_3-x86_64-isos",
+                      "files": [
+                        {
+                          "filename": "test-product-amd-1.3-1732045201-x86_64.raw",
+                          "source": "disk.raw"
+                        }
+                      ],
+                      "version": "1.3"
+                    }
+                  }
+                ]
+              }
+              EOF
+
+              cat > "$(workspaces.data.path)/data.json" << EOF
+              {
+                "mapping": {
+                  "productInfo": {
+                    "filePrefix": "test-product-amd-1.3",
+                    "productCode": "TESTPRDOCUT",
+                    "productName": "Test Product",
+                    "productVersionName": "1.3"
+                  },
+                  "starmap": [
+                    {
+                      "cloud": "aws",
+                      "mappings": {
+                        "aws-emea": {
+                          "destinations": [
+                            {
+                              "architecture": "x86_64",
+                              "destination": "00000000-0000-0000-0000-000000000000",
+                              "overwrite": false,
+                              "restrict_version": true
+                            }
+                          ],
+                          "provider": null
+                        },
+                        "aws-na": {
+                          "destinations": [
+                            {
+                              "architecture": "x86_64",
+                              "destination": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+                              "overwrite": false,
+                              "restrict_version": true
+                            }
+                          ],
+                          "provider": null
+                        }
+                      },
+                      "name": "test-product",
+                      "workflow": "stratosphere"
+                    }
+                  ],
+                  "components": [
+                    {
+                      "name": "MyComponentName"
+                    }
+                  ]
+                }
+              }
+              EOF
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+
+    - name: run-task
+      taskRef:
+        name: marketplaces-push-disk-images
+      params:
+        - name: snapshotPath
+          value: "snapshot_spec.json"
+        - name: dataPath
+          value: "data.json"
+        - name: cloudMarketplacesSecret
+          value: test-secret
+      runAfter:
+        - setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+
+    - name: check-result
+      params:
```

Comment:
no need for param anymore (and it doesn't exist)

Reply:
Oh, that's right! I got rid of it, thanks!